### PR TITLE
Added RabbitMQ Chart

### PR DESCRIPTION
**Pull request by Fortiz2305**

### CODE REVIEW

File: rabbitmq/manifests/rabbitmq-rc.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+metadata:
+  name: rabbitmq
+  labels:
+    app: rabbitmq
+    provider: rabbitmq-server
+    heritage: bitnami
+spec:
+  replicas: 1
+  selector:
+    app: rabbitmq
```
Comment (sameersbn) [inline rabbitmq/manifests/rabbitmq-rc.yaml]: @Fortiz2305 in service images we do not define the `app` label. Only the `provider` label is defined and this case should be `provider: rabbitmq`

f.e. https://github.com/bitnami/charts/blob/master/mariadb/manifests/mariadb-rc.yaml#L16-L19

File: rabbitmq/tpl/rabbitmq-rc.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          value: {{ .rabbitmqClusternodename | quote }}
+        - name: RABBITMQ_VHOST
+          value: {{ .rabbitmqVhost | quote }}
+        - name: RABBITMQ_MANAGERBINDIP
+          value: {{ .rabbitmqManagerbindip | quote }}
+        ports:
+        - name: rabbitmq
+          containerPort: 15672
+        livenessProbe:
+          httpGet:
```
Comment (sameersbn) [inline rabbitmq/tpl/rabbitmq-rc.yaml]: @Fortiz2305 I wonder if `rabbitmqctl status` is a better check for the liveness and readiness probes. I'm not a rabbitmq expert, so I may be wrong here

Comment (Fortiz2305) [inline rabbitmq/tpl/rabbitmq-rc.yaml]: Yes, you are not wrong! I will change it

**Comment by sameersbn** [comment]
LGTM

**Pull request merged**